modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Decoder.scala

```diff
@@ -285,6 +292,26 @@ trait StandardDecoderInstances { this: PrimitiveDecoderInstances =>
         case Right(a) => Right(Left(a))
       }
     }
+
+  implicit final def graphDecodeNonEmptySet[A: Decoder: Order]: Decoder[NonEmptySet[A]] =
+    graphDecodeSet[A].emap { set =>
+      set.headOption match {
+        case Some(head) => Right(NonEmptySet(head, SortedSet(set.drop(1).toSeq: _*)))
```

Comment:
why a SortedSet? can it be like this?
```
set.headOption match {
  case Some(head) => Right(NonEmptySet(head, set - head))
  ...
}
```

Reply:
`NonEmptySet` implementation is a `SortedSet` implementation